cmd/juju/model/dump.go

```diff
@@ -101,6 +101,13 @@ func (c *dumpCommand) Run(ctx *cmd.Context) error {
 	}
 
 	modelDetails, err := store.ModelByName(controllerName, c.model)
```

Comment:
can you please instead add a new method to ControllerCommandBase?

```go
func (*ControllerCommandBase) ModelDetails(modelName string) (*jujuclient.ModelDetails, error)
```

which will do the refresh, just like ModelCommandBase.ModelDetails

Reply:
I am not sure if it's worth it.
It looks like these 2 are the only commands that will call it and I agree with Roger that they both should be model-type commands anyway. I might just change them to be :) This will eliminate code duplication and couple of todos :D

Reply:
I concur. Please notify the mailing lists after it's done.